watson-developer-cloud/discovery-components
# feat: limit response size

#### What do these changes do/fix?

Update `SearchResults` component to update the search parameters and limit the size of the response, to only what is needed to render `SearchResults`.
Update `fetchDocuments()` to take collection ID argument. Fix how response is merged into search store response.
Fix some eslint warnings.

#### How do you test/verify these changes?

Run example app and run a blank search. Note the size of the `/query` response before and after. Ensure that loading a document shows all document data.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?

Yes. See comments below.
